### DISCUSSION TITLE
Bootstrap 5 navbar not properly aligned.

### DISCUSSION THREAD

**Discussion by Srishanth57** [twbs/bootstrap#41056] [Q&A]
While using React, I tried to import the navbar from Bootstrap 5, but the search icon and all were not properly aligned. Please tell me how to fix it!.

**Comment by julien-deramond** [comment]
@Srishanth57 Could you share some code or a reproducible environment, such as a StackBlitz project? You can start with one of the reusable examples available at https://github.com/twbs/examples. This will help us better understand the issue and provide more effective assistance.

**Comment by mchubbit** [comment]
Make sure you're importing the Bootstrap CSS correctly and that all your markup follows the Bootstrap 5 navbar structure. Then, you can try adding the d-flex and align-items-center classes to the navbar elements that need to be aligned. This will vertically center the content within the navbar.
